smoketest/scripts/cli/test_interfaces_bridge.py

```diff
@@ -78,6 +82,95 @@ def test_add_remove_member(self):
             self.session.delete(self._base_path + [interface, 'member'])
 
         self.session.commit()
+    
+    def test_vlan_filter(self):
+        """ Add member interface to bridge and set VLAN filter """
+        for interface in self._interfaces:
+            base = self._base_path + [interface]
+            self.session.set(base + ['address', '192.0.2.1/24'])
+            self.session.set(base + ['vif', '2','address','192.0.3.1/24'])
+
+            vlan_id = 101
+            # assign members to bridge interface
+            for member in self._members:
+                base_member = base + ['member', 'interface', member]
+                self.session.set(base_member + ['allowed-vlan', '2'])
```

Comment:
Please assign the allowed-vlan range and native vlan to a variable which is referenced in the validation logic below.
Using this makes it easier to extend ranges or adding additional ranges via loops.